ykcompile/src/lib.rs

```diff
@@ -362,22 +354,16 @@ impl<TT> TraceCompiler<TT> {
                     let v1 = c_val as u32 as i32;
                     let v2 = (c_val >> 32) as u32 as i32;
                     dynasm!(self.asm
-                        ; mov DWORD [rbp-offset], v1
-                        ; mov DWORD [rbp-offset+4], v2
+                        ; mov DWORD [Rq(ro.reg) + ro.offs], v1
+                        ; mov DWORD [Rq(ro.reg) + ro.offs + 4], v2
```

Comment:
I think we need to be careful here. This is still correct for stack locations. But I don't think it is for `Arg`s, though I'll have to think a bit about what this should be instead.

Reply:
Are you sure it's incorrect? I've just scribbled stack diagrams on paper and I think it remains correct.

E.g. writing two words at rbp-8 and rdi+0:

```
Stack                              Arg

+--------+  rbp                   +---------+  rdi + 8
|write2  |                        |write2   |
|        |                        |         |
+--------+  rbp - 4               +---------+  rdi + 4
|write1  |                        |write1   |
|        |                        |         |
+--------+  rbp - 8               +---------+  rdi
```

In both cases, adding 4 seems correct. Right?

Reply:
You're right!